libraries/provider_component_runit_supervisor_systemd.rb

```diff
@@ -23,6 +23,15 @@ class Systemd < Chef::Provider::LWRPBase
             source "runsvdir-start.service.erb"
           end
 
+          # This cookbook originally installed its unit files in /usr/lib/systemd/system.
+          execute "echo" do
```

Comment:
I'm happy with this change, but like what @srenatus has pointed out here. The name of echo seems a bit odd.